Makefile

```diff
@@ -18,10 +18,10 @@ BUILD_COMMIT := $(shell ./build/get-build-commit.sh)
 BUILD_TIMESTAMP := $(shell ./build/get-build-timestamp.sh)
 BUILD_HOSTNAME := $(shell ./build/get-build-hostname.sh)
 
-LDFLAGS := "-X github.com/open-policy-agent/gatekeeper/version.Version=$(VERSION) \
-	-X github.com/open-policy-agent/gatekeeper/version.Vcs=$(BUILD_COMMIT) \
-	-X github.com/open-policy-agent/gatekeeper/version.Timestamp=$(BUILD_TIMESTAMP) \
-	-X github.com/open-policy-agent/gatekeeper/version.Hostname=$(BUILD_HOSTNAME)"
+LDFLAGS := "-X github.com/open-policy-agent/gatekeeper/pkg/version.Version=$(VERSION) \
```

Comment:
Looks like `LDFLAGS` are not passed into Dockerfile. version, vcs and timestamp will be empty
https://github.com/open-policy-agent/gatekeeper/blob/master/Dockerfile#L28

Reply:
Updated. I kept the defaulting logic for LDFLAGS to just the version.